[debug] [2023-10-23T18:50:06.157Z] ----------------------------------------------------------------------
[debug] [2023-10-23T18:50:06.159Z] Command:       /home/diego/.nvm/versions/node/v20.8.1/bin/node /home/diego/.nvm/versions/node/v20.8.1/bin/firebase deploy
[debug] [2023-10-23T18:50:06.159Z] CLI Version:   12.7.0
[debug] [2023-10-23T18:50:06.159Z] Platform:      linux
[debug] [2023-10-23T18:50:06.159Z] Node Version:  v20.8.1
[debug] [2023-10-23T18:50:06.159Z] Time:          Mon Oct 23 2023 13:50:06 GMT-0500 (Colombia Standard Time)
[debug] [2023-10-23T18:50:06.159Z] ----------------------------------------------------------------------
[debug] 
[debug] [2023-10-23T18:50:06.204Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2023-10-23T18:50:06.204Z] > authorizing via signed-in user (diegopolodev@gmail.com)
[debug] [2023-10-23T18:50:06.204Z] [iam] checking project graphicweb-2d251 for permissions ["firebase.projects.get","firebasehosting.sites.update"]
[debug] [2023-10-23T18:50:06.205Z] > refreshing access token with scopes: []
[debug] [2023-10-23T18:50:06.206Z] >>> [apiv2][query] POST https://www.googleapis.com/oauth2/v3/token [none]
[debug] [2023-10-23T18:50:06.206Z] >>> [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2023-10-23T18:50:06.624Z] <<< [apiv2][status] POST https://www.googleapis.com/oauth2/v3/token 200
[debug] [2023-10-23T18:50:06.624Z] <<< [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2023-10-23T18:50:06.636Z] >>> [apiv2][query] POST https://cloudresourcemanager.googleapis.com/v1/projects/graphicweb-2d251:testIamPermissions [none]
[debug] [2023-10-23T18:50:06.636Z] >>> [apiv2][(partial)header] POST https://cloudresourcemanager.googleapis.com/v1/projects/graphicweb-2d251:testIamPermissions x-goog-quota-user=projects/graphicweb-2d251
[debug] [2023-10-23T18:50:06.636Z] >>> [apiv2][body] POST https://cloudresourcemanager.googleapis.com/v1/projects/graphicweb-2d251:testIamPermissions {"permissions":["firebase.projects.get","firebasehosting.sites.update"]}
[debug] [2023-10-23T18:50:07.388Z] <<< [apiv2][status] POST https://cloudresourcemanager.googleapis.com/v1/projects/graphicweb-2d251:testIamPermissions 200
[debug] [2023-10-23T18:50:07.389Z] <<< [apiv2][body] POST https://cloudresourcemanager.googleapis.com/v1/projects/graphicweb-2d251:testIamPermissions {"permissions":["firebase.projects.get","firebasehosting.sites.update"]}
[debug] [2023-10-23T18:50:07.390Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects/graphicweb-2d251 [none]
[debug] [2023-10-23T18:50:07.900Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects/graphicweb-2d251 200
[debug] [2023-10-23T18:50:07.900Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects/graphicweb-2d251 {"projectId":"graphicweb-2d251","projectNumber":"216355403940","displayName":"graphicweb","name":"projects/graphicweb-2d251","resources":{"hostingSite":"graphicweb-2d251"},"state":"ACTIVE","etag":"1_9f8f97a7-aa5e-4dee-8d4d-f274d5bccf6a"}
[info] 
   Thank you for trying our early preview of Next.js support on Firebase Hosting.
   During the preview, support is best-effort and breaking changes can be expected. Proceed with caution.

   Documentation: https://firebase.google.com/docs/hosting/frameworks/nextjs
   File a bug: https://github.com/firebase/firebase-tools/issues/new?template=bug_report.md
   Submit a feature request: https://github.com/firebase/firebase-tools/issues/new?template=feature_request.md

   We'd love to learn from you. Express your interest in helping us shape the future of Firebase Hosting: https://goo.gle/41enW5X

[info]    Creating an optimized production build ...

[info] FetchError: request to https://fonts.gstatic.com/s/poppins/v20/pxiDyp8kv8JHgFVrJJLm21lVGdeOYktMqlap.woff2 failed, reason: 
    at ClientRequest.<anonymous> (/home/diego/graphicweb/node_modules/next/dist/compiled/node-fetch/index.js:1:65756)
    at ClientRequest.emit (node:events:514:28)
    at ClientRequest.emit (node:domain:488:12)
    at TLSSocket.socketErrorListener (node:_http_client:495:9)
    at TLSSocket.emit (node:events:514:28)
    at TLSSocket.emit (node:domain:488:12)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at processTicksAndRejections (node:internal/process/task_queues:82:21)
    at runNextTicks (node:internal/process/task_queues:64:3) {
  type: 'system',
  errno: 'ETIMEDOUT',
  code: 'ETIMEDOUT'
}

[info] FetchError: request to https://fonts.gstatic.com/s/poppins/v20/pxiDyp8kv8JHgFVrJJLm21lVF9eOYktMqg.woff2 failed, reason: 
    at ClientRequest.<anonymous> (/home/diego/graphicweb/node_modules/next/dist/compiled/node-fetch/index.js:1:65756)
    at ClientRequest.emit (node:events:514:28)
    at ClientRequest.emit (node:domain:488:12)
    at TLSSocket.socketErrorListener (node:_http_client:495:9)
    at TLSSocket.emit (node:events:514:28)
    at TLSSocket.emit (node:domain:488:12)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at processTicksAndRejections (node:internal/process/task_queues:82:21)
    at runNextTicks (node:internal/process/task_queues:64:3) {
  type: 'system',
  errno: 'ETIMEDOUT',
  code: 'ETIMEDOUT'
}

[info] FetchError: request to https://fonts.gstatic.com/s/poppins/v20/pxiGyp8kv8JHgFVrJJLucXtAOvWDSHFF.woff2 failed, reason: 
    at ClientRequest.<anonymous> (/home/diego/graphicweb/node_modules/next/dist/compiled/node-fetch/index.js:1:65756)
    at ClientRequest.emit (node:events:514:28)
    at ClientRequest.emit (node:domain:488:12)
    at TLSSocket.socketErrorListener (node:_http_client:495:9)
    at TLSSocket.emit (node:events:514:28)
    at TLSSocket.emit (node:domain:488:12)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at processTicksAndRejections (node:internal/process/task_queues:82:21)
    at runNextTicks (node:internal/process/task_queues:64:3) {
  type: 'system',
  errno: 'ETIMEDOUT',
  code: 'ETIMEDOUT'
}

[info] FetchError: request to https://fonts.gstatic.com/s/poppins/v20/pxiGyp8kv8JHgFVrJJLufntAOvWDSHFF.woff2 failed, reason: 
    at ClientRequest.<anonymous> (/home/diego/graphicweb/node_modules/next/dist/compiled/node-fetch/index.js:1:65756)
    at ClientRequest.emit (node:events:514:28)
    at ClientRequest.emit (node:domain:488:12)
    at TLSSocket.socketErrorListener (node:_http_client:495:9)
    at TLSSocket.emit (node:events:514:28)
    at TLSSocket.emit (node:domain:488:12)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at processTicksAndRejections (node:internal/process/task_queues:82:21)
    at runNextTicks (node:internal/process/task_queues:64:3) {
  type: 'system',
  errno: 'ETIMEDOUT',
  code: 'ETIMEDOUT'
}

[info] FetchError: request to https://fonts.gstatic.com/s/poppins/v20/pxiDyp8kv8JHgFVrJJLmg1hVFteOYktMqlap.woff2 failed, reason: 
    at ClientRequest.<anonymous> (/home/diego/graphicweb/node_modules/next/dist/compiled/node-fetch/index.js:1:65756)
    at ClientRequest.emit (node:events:514:28)
    at ClientRequest.emit (node:domain:488:12)
    at TLSSocket.socketErrorListener (node:_http_client:495:9)
    at TLSSocket.emit (node:events:514:28)
    at TLSSocket.emit (node:domain:488:12)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at processTicksAndRejections (node:internal/process/task_queues:82:21)
    at runNextTicks (node:internal/process/task_queues:64:3) {
  type: 'system',
  errno: 'ETIMEDOUT',
  code: 'ETIMEDOUT'
}

[info] FetchError: request to https://fonts.gstatic.com/s/poppins/v20/pxiDyp8kv8JHgFVrJJLmg1hVGdeOYktMqlap.woff2 failed, reason: 
    at ClientRequest.<anonymous> (/home/diego/graphicweb/node_modules/next/dist/compiled/node-fetch/index.js:1:65756)
    at ClientRequest.emit (node:events:514:28)
    at ClientRequest.emit (node:domain:488:12)
    at TLSSocket.socketErrorListener (node:_http_client:495:9)
    at TLSSocket.emit (node:events:514:28)
    at TLSSocket.emit (node:domain:488:12)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at processTicksAndRejections (node:internal/process/task_queues:82:21)
    at runNextTicks (node:internal/process/task_queues:64:3) {
  type: 'system',
  errno: 'ETIMEDOUT',
  code: 'ETIMEDOUT'
}

[info] FetchError: request to https://fonts.gstatic.com/s/poppins/v20/pxiDyp8kv8JHgFVrJJLmg1hVF9eOYktMqg.woff2 failed, reason: 
    at ClientRequest.<anonymous> (/home/diego/graphicweb/node_modules/next/dist/compiled/node-fetch/index.js:1:65756)
    at ClientRequest.emit (node:events:514:28)
    at ClientRequest.emit (node:domain:488:12)
    at TLSSocket.socketErrorListener (node:_http_client:495:9)
    at TLSSocket.emit (node:events:514:28)
    at TLSSocket.emit (node:domain:488:12)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at processTicksAndRejections (node:internal/process/task_queues:82:21)
    at runNextTicks (node:internal/process/task_queues:64:3) {
  type: 'system',
  errno: 'ETIMEDOUT',
  code: 'ETIMEDOUT'
}

[info] FetchError: request to https://fonts.gstatic.com/s/poppins/v20/pxiDyp8kv8JHgFVrJJLmr19VFteOYktMqlap.woff2 failed, reason: 
    at ClientRequest.<anonymous> (/home/diego/graphicweb/node_modules/next/dist/compiled/node-fetch/index.js:1:65756)
    at ClientRequest.emit (node:events:514:28)
    at ClientRequest.emit (node:domain:488:12)
    at TLSSocket.socketErrorListener (node:_http_client:495:9)
    at TLSSocket.emit (node:events:514:28)
    at TLSSocket.emit (node:domain:488:12)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at processTicksAndRejections (node:internal/process/task_queues:82:21)
    at runNextTicks (node:internal/process/task_queues:64:3) {
  type: 'system',
  errno: 'ETIMEDOUT',
  code: 'ETIMEDOUT'
}

[info] FetchError: request to https://fonts.gstatic.com/s/poppins/v20/pxiDyp8kv8JHgFVrJJLmr19VGdeOYktMqlap.woff2 failed, reason: 
    at ClientRequest.<anonymous> (/home/diego/graphicweb/node_modules/next/dist/compiled/node-fetch/index.js:1:65756)
    at ClientRequest.emit (node:events:514:28)
    at ClientRequest.emit (node:domain:488:12)
    at TLSSocket.socketErrorListener (node:_http_client:495:9)
    at TLSSocket.emit (node:events:514:28)
    at TLSSocket.emit (node:domain:488:12)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at processTicksAndRejections (node:internal/process/task_queues:82:21)
    at runNextTicks (node:internal/process/task_queues:64:3) {
  type: 'system',
  errno: 'ETIMEDOUT',
  code: 'ETIMEDOUT'
}

[info] FetchError: request to https://fonts.gstatic.com/s/poppins/v20/pxiDyp8kv8JHgFVrJJLmr19VF9eOYktMqg.woff2 failed, reason: 
    at ClientRequest.<anonymous> (/home/diego/graphicweb/node_modules/next/dist/compiled/node-fetch/index.js:1:65756)
    at ClientRequest.emit (node:events:514:28)
    at ClientRequest.emit (node:domain:488:12)
    at TLSSocket.socketErrorListener (node:_http_client:495:9)
    at TLSSocket.emit (node:events:514:28)
    at TLSSocket.emit (node:domain:488:12)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at processTicksAndRejections (node:internal/process/task_queues:82:21)
    at runNextTicks (node:internal/process/task_queues:64:3) {
  type: 'system',
  errno: 'ETIMEDOUT',
  code: 'ETIMEDOUT'
}

[info] FetchError: request to https://fonts.gstatic.com/s/poppins/v20/pxiDyp8kv8JHgFVrJJLmy15VFteOYktMqlap.woff2 failed, reason: 
    at ClientRequest.<anonymous> (/home/diego/graphicweb/node_modules/next/dist/compiled/node-fetch/index.js:1:65756)
    at ClientRequest.emit (node:events:514:28)
    at ClientRequest.emit (node:domain:488:12)
    at TLSSocket.socketErrorListener (node:_http_client:495:9)
    at TLSSocket.emit (node:events:514:28)
    at TLSSocket.emit (node:domain:488:12)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at processTicksAndRejections (node:internal/process/task_queues:82:21)
    at runNextTicks (node:internal/process/task_queues:64:3) {
  type: 'system',
  errno: 'ETIMEDOUT',
  code: 'ETIMEDOUT'
}

[info] FetchError: request to https://fonts.gstatic.com/s/poppins/v20/pxiDyp8kv8JHgFVrJJLmy15VGdeOYktMqlap.woff2 failed, reason: 
    at ClientRequest.<anonymous> (/home/diego/graphicweb/node_modules/next/dist/compiled/node-fetch/index.js:1:65756)
    at ClientRequest.emit (node:events:514:28)
    at ClientRequest.emit (node:domain:488:12)
    at TLSSocket.socketErrorListener (node:_http_client:495:9)
    at TLSSocket.emit (node:events:514:28)
    at TLSSocket.emit (node:domain:488:12)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at processTicksAndRejections (node:internal/process/task_queues:82:21)
    at runNextTicks (node:internal/process/task_queues:64:3) {
  type: 'system',
  errno: 'ETIMEDOUT',
  code: 'ETIMEDOUT'
}

[info] FetchError: request to https://fonts.gstatic.com/s/poppins/v20/pxiDyp8kv8JHgFVrJJLmy15VF9eOYktMqg.woff2 failed, reason: 
    at ClientRequest.<anonymous> (/home/diego/graphicweb/node_modules/next/dist/compiled/node-fetch/index.js:1:65756)
    at ClientRequest.emit (node:events:514:28)
    at ClientRequest.emit (node:domain:488:12)
    at TLSSocket.socketErrorListener (node:_http_client:495:9)
    at TLSSocket.emit (node:events:514:28)
    at TLSSocket.emit (node:domain:488:12)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at processTicksAndRejections (node:internal/process/task_queues:82:21)
    at runNextTicks (node:internal/process/task_queues:64:3) {
  type: 'system',
  errno: 'ETIMEDOUT',
  code: 'ETIMEDOUT'
}

[info] FetchError: request to https://fonts.gstatic.com/s/poppins/v20/pxiByp8kv8JHgFVrLDz8Z11lFd2JQEl8qw.woff2 failed, reason: 
    at ClientRequest.<anonymous> (/home/diego/graphicweb/node_modules/next/dist/compiled/node-fetch/index.js:1:65756)
    at ClientRequest.emit (node:events:514:28)
    at ClientRequest.emit (node:domain:488:12)
    at TLSSocket.socketErrorListener (node:_http_client:495:9)
    at TLSSocket.emit (node:events:514:28)
    at TLSSocket.emit (node:domain:488:12)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at processTicksAndRejections (node:internal/process/task_queues:82:21)
    at runNextTicks (node:internal/process/task_queues:64:3) {
  type: 'system',
  errno: 'ETIMEDOUT',
  code: 'ETIMEDOUT'
}

[info] FetchError: request to https://fonts.gstatic.com/s/poppins/v20/pxiByp8kv8JHgFVrLDz8Z1JlFd2JQEl8qw.woff2 failed, reason: 
    at ClientRequest.<anonymous> (/home/diego/graphicweb/node_modules/next/dist/compiled/node-fetch/index.js:1:65756)
    at ClientRequest.emit (node:events:514:28)
    at ClientRequest.emit (node:domain:488:12)
    at TLSSocket.socketErrorListener (node:_http_client:495:9)
    at TLSSocket.emit (node:events:514:28)
    at TLSSocket.emit (node:domain:488:12)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at processTicksAndRejections (node:internal/process/task_queues:82:21)
    at runNextTicks (node:internal/process/task_queues:64:3) {
  type: 'system',
  errno: 'ETIMEDOUT',
  code: 'ETIMEDOUT'
}

[info] FetchError: request to https://fonts.gstatic.com/s/poppins/v20/pxiByp8kv8JHgFVrLDz8Z1xlFd2JQEk.woff2 failed, reason: 
    at ClientRequest.<anonymous> (/home/diego/graphicweb/node_modules/next/dist/compiled/node-fetch/index.js:1:65756)
    at ClientRequest.emit (node:events:514:28)
    at ClientRequest.emit (node:domain:488:12)
    at TLSSocket.socketErrorListener (node:_http_client:495:9)
    at TLSSocket.emit (node:events:514:28)
    at TLSSocket.emit (node:domain:488:12)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at processTicksAndRejections (node:internal/process/task_queues:82:21)
    at runNextTicks (node:internal/process/task_queues:64:3) {
  type: 'system',
  errno: 'ETIMEDOUT',
  code: 'ETIMEDOUT'
}

[info] FetchError: request to https://fonts.gstatic.com/s/poppins/v20/pxiEyp8kv8JHgFVrJJbecnFHGPezSQ.woff2 failed, reason: 
    at ClientRequest.<anonymous> (/home/diego/graphicweb/node_modules/next/dist/compiled/node-fetch/index.js:1:65756)
    at ClientRequest.emit (node:events:514:28)
    at ClientRequest.emit (node:domain:488:12)
    at TLSSocket.socketErrorListener (node:_http_client:495:9)
    at TLSSocket.emit (node:events:514:28)
    at TLSSocket.emit (node:domain:488:12)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at processTicksAndRejections (node:internal/process/task_queues:82:21)
    at runNextTicks (node:internal/process/task_queues:64:3) {
  type: 'system',
  errno: 'ETIMEDOUT',
  code: 'ETIMEDOUT'
}

[info] FetchError: request to https://fonts.gstatic.com/s/poppins/v20/pxiEyp8kv8JHgFVrJJnecnFHGPezSQ.woff2 failed, reason: 
    at ClientRequest.<anonymous> (/home/diego/graphicweb/node_modules/next/dist/compiled/node-fetch/index.js:1:65756)
    at ClientRequest.emit (node:events:514:28)
    at ClientRequest.emit (node:domain:488:12)
    at TLSSocket.socketErrorListener (node:_http_client:495:9)
    at TLSSocket.emit (node:events:514:28)
    at TLSSocket.emit (node:domain:488:12)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at processTicksAndRejections (node:internal/process/task_queues:82:21)
    at runNextTicks (node:internal/process/task_queues:64:3) {
  type: 'system',
  errno: 'ETIMEDOUT',
  code: 'ETIMEDOUT'
}

[info] FetchError: request to https://fonts.gstatic.com/s/poppins/v20/pxiEyp8kv8JHgFVrJJfecnFHGPc.woff2 failed, reason: 
    at ClientRequest.<anonymous> (/home/diego/graphicweb/node_modules/next/dist/compiled/node-fetch/index.js:1:65756)
    at ClientRequest.emit (node:events:514:28)
    at ClientRequest.emit (node:domain:488:12)
    at TLSSocket.socketErrorListener (node:_http_client:495:9)
    at TLSSocket.emit (node:events:514:28)
    at TLSSocket.emit (node:domain:488:12)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
  type: 'system',
  errno: 'ETIMEDOUT',
  code: 'ETIMEDOUT'
}

[info] Failed to compile.

src/app/layout.tsx
`next/font` error:
Failed to fetch `Poppins` from Google Fonts.

src/app/layout.tsx
An error occured in `next/font`.

NextFontError: Failed to fetch `Poppins` from Google Fonts.
    at nextFontError (/home/diego/graphicweb/node_modules/next/dist/compiled/@next/font/dist/next-font-error.js:8:17)
    at /home/diego/graphicweb/node_modules/next/dist/compiled/@next/font/dist/google/loader.js:110:53
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at runNextTicks (node:internal/process/task_queues:64:3)
    at listOnTimeout (node:internal/timers:540:9)
    at process.processTimers (node:internal/timers:514:7)
    at async Promise.all (index 1)
    at async nextFontGoogleFontLoader (/home/diego/graphicweb/node_modules/next/dist/compiled/@next/font/dist/google/loader.js:94:33)
    at async Span.traceAsyncFn (/home/diego/graphicweb/node_modules/next/dist/trace/trace.js:105:20)
    at async /home/diego/graphicweb/node_modules/next/dist/build/webpack/loaders/next-font-loader/index.js:76:87
    at tryRunOrWebpackError (/home/diego/graphicweb/node_modules/next/dist/compiled/webpack/bundle5.js:28:311563)
    at __webpack_require_module__ (/home/diego/graphicweb/node_modules/next/dist/compiled/webpack/bundle5.js:28:131195)
    at __nested_webpack_require_153754__ (/home/diego/graphicweb/node_modules/next/dist/compiled/webpack/bundle5.js:28:130634)
    at /home/diego/graphicweb/node_modules/next/dist/compiled/webpack/bundle5.js:28:131487
    at symbolIterator (/home/diego/graphicweb/node_modules/next/dist/compiled/neo-async/async.js:1:14444)
    at done (/home/diego/graphicweb/node_modules/next/dist/compiled/neo-async/async.js:1:14824)
    at Hook.eval [as callAsync] (eval at create (/home/diego/graphicweb/node_modules/next/dist/compiled/webpack/bundle5.js:13:28867), <anonymous>:15:1)
    at /home/diego/graphicweb/node_modules/next/dist/compiled/webpack/bundle5.js:28:130354
    at symbolIterator (/home/diego/graphicweb/node_modules/next/dist/compiled/neo-async/async.js:1:14402)
    at timesSync (/home/diego/graphicweb/node_modules/next/dist/compiled/neo-async/async.js:1:5027)
-- inner error --
NextFontError: Failed to fetch `Poppins` from Google Fonts.
    at nextFontError (/home/diego/graphicweb/node_modules/next/dist/compiled/@next/font/dist/next-font-error.js:8:17)
    at /home/diego/graphicweb/node_modules/next/dist/compiled/@next/font/dist/google/loader.js:110:53
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at runNextTicks (node:internal/process/task_queues:64:3)
    at listOnTimeout (node:internal/timers:540:9)
    at process.processTimers (node:internal/timers:514:7)
    at async Promise.all (index 1)
    at async nextFontGoogleFontLoader (/home/diego/graphicweb/node_modules/next/dist/compiled/@next/font/dist/google/loader.js:94:33)
    at async Span.traceAsyncFn (/home/diego/graphicweb/node_modules/next/dist/trace/trace.js:105:20)
    at async /home/diego/graphicweb/node_modules/next/dist/build/webpack/loaders/next-font-loader/index.js:76:87
    at Object.<anonymous> (/home/diego/graphicweb/node_modules/next/dist/build/webpack/loaders/css-loader/src/index.js??ruleSet[1].rules[12].oneOf[2].use[1]!/home/diego/graphicweb/node_modules/next/dist/build/webpack/loaders/next-font-loader/index.js??ruleSet[1].rules[12].oneOf[2].use[2]!/home/diego/graphicweb/node_modules/next/font/google/target.css?{"path":"src/app/layout.tsx","import":"Poppins","arguments":[{"display":"swap","weight":["300","400","500","600","700"],"style":["normal","italic"],"subsets":["latin-ext"]}],"variableName":"poppins"}:1:7)
    at /home/diego/graphicweb/node_modules/next/dist/compiled/webpack/bundle5.js:28:915116
    at Hook.eval [as call] (eval at create (/home/diego/graphicweb/node_modules/next/dist/compiled/webpack/bundle5.js:13:28645), <anonymous>:7:1)
    at /home/diego/graphicweb/node_modules/next/dist/compiled/webpack/bundle5.js:28:131228
    at tryRunOrWebpackError (/home/diego/graphicweb/node_modules/next/dist/compiled/webpack/bundle5.js:28:311517)
    at __webpack_require_module__ (/home/diego/graphicweb/node_modules/next/dist/compiled/webpack/bundle5.js:28:131195)
    at __nested_webpack_require_153754__ (/home/diego/graphicweb/node_modules/next/dist/compiled/webpack/bundle5.js:28:130634)
    at /home/diego/graphicweb/node_modules/next/dist/compiled/webpack/bundle5.js:28:131487
    at symbolIterator (/home/diego/graphicweb/node_modules/next/dist/compiled/neo-async/async.js:1:14444)
    at done (/home/diego/graphicweb/node_modules/next/dist/compiled/neo-async/async.js:1:14824)

Generated code for /home/diego/graphicweb/node_modules/next/dist/build/webpack/loaders/css-loader/src/index.js??ruleSet[1].rules[12].oneOf[2].use[1]!/home/diego/graphicweb/node_modules/next/dist/build/webpack/loaders/next-font-loader/index.js??ruleSet[1].rules[12].oneOf[2].use[2]!/home/diego/graphicweb/node_modules/next/font/google/target.css?{"path":"src/app/layout.tsx","import":"Poppins","arguments":[{"display":"swap","weight":["300","400","500","600","700"],"style":["normal","italic"],"subsets":["latin-ext"]}],"variableName":"poppins"}


[info] 
> Build failed because of webpack errors

[debug] [2023-10-23T18:50:16.729Z] Error: ENOENT: no such file or directory, open '/home/diego/graphicweb/.next/export-marker.json'
[error] 
[error] Error: An unexpected error has occurred.
